libs/cassandra-util/src/Cassandra/Util.hs

```diff
@@ -44,3 +44,37 @@ defInitCassandra ks h p lg =
       . setContacts (unpack h) []
       . setKeyspace (Keyspace ks)
     $ defSettings
+
+-- | Read cassandra's writetimes https://docs.datastax.com/en/dse/5.1/cql/cql/cql_using/useWritetime.html
+-- as UTCTime values without any loss of precision
+newtype Writetime a = Writetime {writetimeToUTC :: UTCTime}
+
+instance Cql (Writetime a) where
+  ctype = Tagged BigIntColumn
+  toCql = CqlBigInt . writetimeToInt64
+  fromCql (CqlBigInt n) =
```

Comment:
Should we have a roundtrip test or two for these two?

Reply:
The integration test already shows that `toCql . fromCql` is the identity. The other direction (`toCql . fromCql`) we don't need, because we never convert any arbitrary `UTCTime` values to Writetime values, because `Writetime` values always originate in the database.